OJP/OJP_Trips.xsd

```diff
@@ -722,6 +727,25 @@
 			<xs:enumeration value="upAndDown"/>
 			<xs:enumeration value="downAndUp"/>
 		</xs:restriction>
+	</xs:simpleType>
+		<xs:simpleType name="AccessibilityFeaturesTypeEnumeration">
+		<xs:annotation>
+			<xs:documentation>Allowed values for AccessibilityFeatures.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="visualSigns"/>
+			<xs:enumeration value="tactileGuidingStrips"/>
+			<xs:enumeration value="tactileOrAuditorySigns"/>
+			<xs:enumeration value="elevatorAccousticAnnouncements"/>
```

Comment:
why do you restrict to elevator ?... the same could apply to travelator, etc.

Reply:
Acoustic announcements are helpful or even important for the visually impaired - but only in the case of an elevator (to leave on the correct floor), not in the case of a travelator or an escalator. LiftEquipment in NeTEx has an attribute AcousticAnnouncement, other Equipments have not. 
TYPO: A**cc**oustic -> Acoustic

Reply:
fixed acoustic. but not the rest of the definitions

Reply:
@Aurige When ok for you, could you resolve the conversation?